src/app/container/container.component.html

```diff
@@ -14,16 +14,6 @@
   ~    limitations under the License.
   -->
 <div *ngIf="tool; else workflow">
-  <div class="row m-1" *ngIf="error || missingWarning">
```

Comment:
The `error` and `missingWarning` properties do not exist in the controller, so this `*ngIf` never evaluated to true. Just removed the block.